widget_bandsplot/schemas/pdos.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$id": "https://example.com/pdos",
```

Comment:
You could use the GitHub org as a namespace here, similar to [here](https://github.com/aiidalab/aiidalab/blob/37c92fbeee0a9c02b8211e1c1767bb849b84eb8f/aiidalab/registry/schemas/app.schema.json#L3).

Reply:
But the tricky part is we don't have a version released with this file, so I'll use the file URL of the branch. But I can open an issue and change this after a new version with this released.

Reply:
You can interpret this more as a name than an actual URL. Just use the URL it will have after merge.